app/src/androidTest/java/com/github/houseorganizer/houseorganizer/CustomMatchers.java

```diff
@@ -0,0 +1,43 @@
+package com.github.houseorganizer.houseorganizer;
+
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.drawable.Drawable;
+import android.view.View;
+import android.widget.ImageView;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+public class CustomMatchers {
+    public static Matcher<View> withBitmap(final Bitmap expected) {
```

Comment:
This class defines a custom matcher to compare bitmaps is that it ?

Reply:
yes, it compares the view's bitmap (if it is an image) to another given bitmap, i use it to make sure the QR code that is generated and shown is the one with the right household id